src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/UpdateOperationPage.java

```diff
@@ -46,9 +71,9 @@ public CommandResult get() {
     }
   }
 
-  private CommandResult.Error getError(DocumentId documentId, Throwable throwable) {
+  private CommandResult.Error getError(List<DocumentId> documentIds, Throwable throwable) {
     String message =
-        "Failed to update document with _id %s: %s".formatted(documentId, throwable.getMessage());
+        "Failed to update documents with _id %s: %s".formatted(documentIds, throwable.getMessage());
```

Comment:
I think that similar to exception key, formatting of documentIds probably should be a shared utility method since we may want to add quoting/escaping at some point -- start with just `List.toString()` for now, but I have found that it is good to distinguish f.ex between number `1` and String `"1"`... especially as String keys can have all kind of content.

Reply:
Come to think of this, since the only value where simple "value.toString()" may not be sufficient is `String` (where we'd want to surround value in quotes), we could easily change `toString()` method for `DocumentId.StringId` and the message would be changed?

Downside is that there are lots of tests that compare exact exception message.

I can do a PR after merging this one so as not to extend scope here.